bot_count: 3
darp_mode: 3
